src/wazuh_modules/sca/sca_impl/src/sca_policy_parser.hpp

```diff
@@ -36,7 +36,7 @@ class PolicyParser
     /// @brief Constructs a PolicyParser and loads the YAML file.
     /// @param filename Path to the YAML policy file.
     /// @param yamlDocument Optional pointer to an already loaded YAML document.
-    explicit PolicyParser(const std::filesystem::path& filename, std::unique_ptr<IYamlDocument> yamlDocument = nullptr);
+    explicit PolicyParser(const std::filesystem::path& filename, const int commandsTimeout, const bool remoteEnabled, std::unique_ptr<IYamlDocument> yamlDocument = nullptr);
```

Comment:
```suggestion
    explicit PolicyParser(const std::filesystem::path& filename, const int commandsTimeout, const bool commandsEnabled, std::unique_ptr<IYamlDocument> yamlDocument = nullptr);
```

I think what you want to achieve can be done by setting `commandsEnabled` as true when the remote enabled setting is true or the `policy.is_remote` is false.

Reply:
Done